Makefile

```diff
@@ -7,8 +7,6 @@ TMP_GOPATH        ?= /tmp/thanos-go
 GOBIN             ?= ${GOPATH}/bin
 GO111MODULE       ?= on
 export GO111MODULE
-GOPROXY           ?= https://proxy.golang.org
```

Comment:
Why are we removing the proxy here? (:

Reply:
Otherwise we have a failing build, the same reason why we had to update to a newer tsdb here: https://github.com/thanos-io/thanos/pull/1380

But I didn't want to update to a newer tsdb for a patch release.

Reply:
We need that. It's also only for 0.6.1